modules/apps/layout/layout-seo-web-test/src/testIntegration/java/com/liferay/layout/seo/web/internal/servlet/taglib/test/OpenGraphTopHeadDynamicIncludeTest.java

```diff
@@ -295,6 +295,10 @@ public void testIncludeDefaultTitleAndDescription() throws Exception {
 		MockHttpServletResponse mockHttpServletResponse =
 			new MockHttpServletResponse();
 
+		_layout.setDescriptionMap(RandomTestUtil.randomLocaleStringMap());
+
+		_layout = _layoutLocalService.updateLayout(_layout);
+
```

Comment:
shouldn't we do this in the setUp? All tests should have a correctly initialized layout (even if they don't use the description field).

Reply:
perfect, thanks!

I will move that code to the setUp